install.sh

```diff
@@ -329,6 +329,9 @@ nvm_do_install() {
       exit 1
     fi
     install_nvm_as_script
+  else
+    echo >&2 'The environment variable $METHOD is set to "'"${METHOD}"'", which is not recognized as a valid installation method.'
```

Comment:
Why the `"'"` thing, instead of something like this?:
```sh
echo >&2 "The environment variable \$METHOD is set to \"${METHOD}\", which is not recognized as a valid installation method.'
```

Reply:
Personal style. `\"` is fine too. I just use `"'"` by default because `\"` has been difficult for me to get right when doing more complex quotation escapes in Bash.

Reply:
it seems like shellcheck is erroring out on this. can you update it so tests are passing? style choices beyond that can be figured out later :-)

Reply:
If I'm reading it right, this is the cause of the failure: "SC2016: Expressions don't expand in single quotes, use double quotes for that." I think getting rid of the single quotes like you suggest will fix that, so I'll give it a try.